### PR TITLE
chore(Python): longest common subsequence

### PR DESCRIPTION
**Pull request by ssaumyaa7**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

Implemented python code to give length of Longest Common Subsequence

**Other information:**
NA

### CODE REVIEW

**Comment by ssaumyaa7** [comment]
@Arsenic-ATG  @atin  @sridhar-5  @cyberwizard1001 

Length of Longest common subsequence in python

**Review by cyberwizard1001 — CHANGES_REQUESTED** [review]
Looked up the code, logic seems fine. However, you have to modify the naming conventions. According to pep-8, classes have continuous words identified by having the first letter as capital (which is fine) but functions are to be fully lowercase with every word separated by an underscore. Can you make sure the code follows convention?

@ming-tsai Is it possible to add this as a check?

**Comment by ssaumyaa7** [comment]
@cyberwizard1001 @ming-tsai is it good to go now ?

**Review by sridhar-5 — APPROVED** [review]
Nice Work @ssaumyaa7 . @ming-tsai we can take it forward.

Thanks

**Review by sridhar-5 — APPROVED** [review]

File: algorithms/Python/README.md [hunk 1]
```diff
@@ -36,3 +36,4 @@
 3. [Word Count](strings/word_count.py)
 4. [Remove Duplicates from a String](strings/remove_duplicates_from_a_string.py)
 5. [First Non Repeating Character](strings/first_non_repeating_character.py)
+6. [Length of Longest Common Subsequence](strings/length_LCS.py)
```
Comment (ming-tsai) [inline algorithms/Python/README.md:39]: Please rename the file name
```suggestion
6. [Longest Common Subsequence](strings/longest_common_subsequence.py)
```

**Comment by ming-tsai** [comment]
Hi @sridhar-5 and @cyberwizard1001,
Please check it again, the last commit of @ssaumyaa7 is only rename the file's name

**Review by sridhar-5 — APPROVED** [review]
Code looks good to me @ming-tsai.
And file name is also following the organization's contribution guide lines I guess

**Review by ming-tsai — APPROVED** [review]

**Comment by sridhar-5** [comment]
Nice Work Guys. @ssaumyaa7 @ming-tsai @cyberwizard1001

**Pull request merged**